rollup.config.js

```diff
@@ -14,7 +14,14 @@ export default [
       {file: pkg.main, format: 'cjs'},
       {file: pkg.module, format: 'es'},
     ],
-    plugins: [ts(), resolve(), babel(), commonjs()],
+    plugins: [
+      ts(),
+      resolve(),
+      babel({
+        extensions: ['.ts', '.tsx'],
```

Comment:
(2) We `allowJS` in our project, so it might be safer to achieve this with some sort of ignore pattern for `node_modules`.

Reply:
Babel will still ignore `node_modules` by default, but I can extend `extensions` to include `js` and `jsx`.

Reply:
I have mixed feeling regarding transpiling down the module version. IMO, publishing only ES5 code is the source of a lot of inefficiencies on the web today.

Reply:
> Babel will still ignore node_modules by default, but I can extend extensions to include js and jsx.

Oh ok great.

> I have mixed feeling regarding transpiling down the module version. IMO, publishing only ES5 code is the source of a lot of inefficiencies on the web today.

Yeah this is a bummer, but I don't think we have much of a choice (don't want users to need to compile node_modules).